library-best-practices/JLBP-16.md

```diff
@@ -0,0 +1,60 @@
+[JLBP-16] Ensure version alignment of dependencies for consumers
+----------------------------------------------------------------
+
+For multi-module projects, this best practice assumes you have already applied
+[JLBP-15](JLBP-15.md), so that your project has a BOM.
+
+### Achieving version alignment
+
+- Version alignment is defined in the [glossary](glossary.md).
+- Version alignment increases the likelihood that consumers' build systems will
+  pick the right versions of direct and transitive dependencies, reducing the
+  number of conflicts.
+- As noted in the definition, to fix a version misalignment caused by a shorter
+  path to a version of a dependency that is not the upper bound in the
+  dependency tree, a direct dependency needs to be added so that Maven consumers
+  will pick the correct version.
+- See the details specific to each build system in the sections below.
+
+### Maven
+
+- Use `requireUpperBoundDeps` enforcement to ensure that you are using the
+  version of each dependency that is the highest in your dependency tree.
```

Comment:
highest version of each dependency since "highest in your dependency tree" makes it sound like it's tree position that matters here.

Reply:
Will fix.